### PR TITLE
chore: use bhFilterToggle on Entry from Transfer

### PR DESCRIPTION
**Pull request by jniles**
This commit makes the findTransfer modal use the `bhFilterToggle` component for filtering.  It also disables the submit button until data is loaded and removes an unnecessary binding.

### CODE REVIEW

File: client/src/modules/stock/entry/modals/findTransfer.modal.js [hunk 1]
```diff
@@ -75,13 +65,12 @@ function StockFindTransferModalController(
   }
 
   function rowSelectionCallback(row) {
-    vm.selectedRow = row.entity;
+    selectedRow = row.entity;
```
Comment (mbayopanda) [inline client/src/modules/stock/entry/modals/findTransfer.modal.js:68]: 👍

**Review by mbayopanda — COMMENTED** [review]
I have a non blocking comment, is it possible that these buttons have the same size ?

Capture d’écran 2020-10-07 à 10 41 28

**Comment by jniles** [comment]
Great idea.  Fixed:

image

**Review by mbayopanda — APPROVED** [review]
LGTM 👍

**Pull request merged**